drivers/include/at86rf2xx.h

```diff
@@ -87,7 +87,13 @@ extern "C" {
 /**
  * @brief   Base (minimal) RSSI value in dBm
  */
-#define RSSI_BASE_VAL                   (-91)
+#if MODULE_AT86RF233
+#   define RSSI_BASE_VAL                   (-94)
+#elif MODULE_AT86RF212B
+#   define RSSI_BASE_VAL                   (-97)
```

Comment:
it seems like this value depends on the PHY mode, according to the data sheet for 212B.
The mode which yields -97 is not settable directly via `ifconfig x set page y`. IMO it's better to default to -98, which is what the modes for page 2 give on both EU and US channel bands.
Also, add a note that the 212B value is valid for page 2 only. The value is -100 or -99 for page 0, depending on channel setting.